vic/vic_run/src/vic_run.c

```diff
@@ -131,6 +131,13 @@ vic_run(force_data_struct   *force,
         gauge_correction[0] = 1;
         gauge_correction[1] = 1;
     }
+
+    // Canopy Iterations
+    if (!options.CLOSE_ENERGY) {
+        // do not iterate to close energy balance
+        param.MAX_ITER_GRND_CANOPY = 0;
+    }
+
```

Comment:
I still don't like this. I get why this can't be in `initialize_parameters.c` but as a general rule, we shouldn't be modifying the parameters structure inside `vic_run`. Can you find a better place for this inside `vic_init`?

Reply:
I agree, I don't like it either but I couldn't think of a better option. I hadn't thought of putting it in `vic_init` since we aren't changing any parameter values in there but I guess we could. 

@jhamman what do you think about only updating (as currently in `vic_run`) when we are not running with `CLOSE_ENERGY` being set to true? I actually don't like that but I can't think of a better way to do it since we've made `MAX_ITER_GRND_CANOPY` a parameter and we don't want to hard code it except in the case of `CLOSE_ENERGY` being false

Reply:
I don't recall the full order of operations but you there should be a straightforward way to do this somewhere within the `vic_init` space. It just needs to be updated after both values are available. Can you look in `vic_init` and tell me at what point they are both populated?

Reply:
I've already updated `vic_init` and am testing it out right now - stand by